appveyor.yml

```diff
@@ -5,6 +5,10 @@ environment:
 
 install:
 
+cache:
+  - nsissetup.exe
+  - NsProcess.zip

```

Comment:
Seems to work fine. :+1: 

I was under the false impression that the fetch script would put all the .dlls in the windows subdirectory. It doesn't, so the fetch script still does more work than necessary. Adding all the other .dlls to the cache would mean you'd have to add a line for every single file of them (wildcards don't seem to be supported), so we'd have to maintain that list in at least three places. I guess we can do without that for now and change it if timeouts while fetching from nuget becomes a problem at some point.